modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/singlecelltracegeneration/v0.1.3.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/singlecelltracegeneration/v0.1.3/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "owl:deprecated": true,
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.3"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SingleCellTraceGenerationShape",
+      "@type": "sh:NodeShape",
+      "label": "Trace generation shape definition",
+      "targetClass": "nsg:SingleCellTraceGeneration",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.3/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:protocolDescription",
```

Comment:
Looking at the properties of that shape I don't think it is relevant in the simulation domain context.

Reply:
I think protocol's infos need to  be register as an entity. Now I agree that the current protocol schema is not relevant in this context. I created an [issue](https://github.com/INCF/neuroshapes/issues/99) so that a generic protocol schema can be created and used here through the singlecelltracegeneration activity.

Reply:
So at the moment I'd prefer to keep it as a generic string. When we'll have some feedback on how the data structure should be formalized I'll comment on the issue you mentioned.

Reply:
A [generic protocol](https://github.com/INCF/neuroshapes/blob/111b8e7e346b66848b0690da2eae69f1914e33b7/modules/core/src/main/resources/schemas/neurosciencegraph/core/protocol/v0.1.4.json) entity is now created and can be used to provide protocol information. If using a protocol is not relevant then then add the description as an activity comment.

Reply:
Well, it is currently in the SingleCellTraceGeneration activity.

Reply:
Removed protocolDescription, will use Activity description as suggested.